### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by smiile8888**
Adding icons and style for them

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,70 +1,91 @@
...
+        </button>
+        <button data-ig="true" class="removeAllBlur button">
+          Rem
+        </button>
+        <!-- Smile Added -->
+        <button id="file-text-icon" class="button">
+          <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="file-alt" role="img"
+            xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class="svg-inline--fa fa-file-alt fa-w-12 fa-3x">
+            <path fill="currentColor"
+              d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm64 236c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12v8zm0-64c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12v8zm0-72v8c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12zm96-114.1v6.1H256V0h6.1c6.4 0 12.5 2.5 17 7l97.9 98c4.5 4.5 7 10.6 7 16.9z"
```
Comment (JaturongKongmanee) [inline index.html:56]: why u need this

File: index.html [hunk 2]
```diff
@@ -1,70 +1,91 @@
...
+        <button data-ig="true" class="removeAllBlur button">
+          Rem
+        </button>
+        <button data-ig="true" class="removeAllBlur button">
+          Rem
+        </button>
+        <!-- Smile Added -->
+        <button id="file-text-icon" class="button">
+          <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="file-alt" role="img"
+            xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class="svg-inline--fa fa-file-alt fa-w-12 fa-3x">
```
Comment (JaturongKongmanee) [inline index.html:54]: how can I deal with these class -> fa fa-file.....

File: index.html (hunk 1 above)
Comment (smiile8888) [reply index.html:56]: Deleted!

File: index.html (hunk 2 above)
Comment (smiile8888) [reply index.html]: Deleted!

**Pull request closed without merging**